test/fixtures/semantic-html5-scenarios/section-with-chapter-signifier.adoc

```diff
@@ -0,0 +1,17 @@
+:doctype: book
+:sectnums:
+:noheader:
+:chapter-signifier: Chapter
+
+[chapter]
+= Part one
```

Comment:
I would prefer fixtures to be well formed and for attribute entries to be below the document title. Here, the part is being reclassified as a chapter, which isn't even technically supported by AsciiDoc. This document should be:

```
= Document Title
:doctype: book
:sectnums:
:noheader:
:chapter-signifier: Chapter

== Introduction

=== Background

==== Problem

== Literature Review

=== Model
```

Reply:
I think my goal was to create an headless document with section level 1. I want fixtures to be as independent as possible. If I add a document title then this test will break if I do some changes on document title (even though sections are fine).

Maybe we should improve the "test engine" in order to validate a given portion of the HTML produced (and not systematically the complete HTML)?

Reply:
> Maybe we should improve the "test engine" in order to validate a given portion of the HTML produced (and not systematically the complete HTML)?

That's one idea. The other is to rely on the noheader attribute for cases like these.

Reply:
> The other is to rely on the noheader attribute for cases like these.

I like the `noheader` approach, let's keep it simple for now 😄